### DISCUSSION TITLE
Filament\FilamentManager::getUserName(): Return value must be of type string, null returned

### DISCUSSION THREAD

**Discussion by devhoussam1998** [filamentphp/filament#10328] [Help]
https://app.test/email-verification/prompt

<img width="1708" alt="Screenshot 2023-12-15 at 13 24 33" src="https://github.com/filamentphp/filament/assets/121255432/79f2dbe7-ca62-4439-97a9-01fe6db71731">

**Comment by semno** [comment]
You must have `name` field in users table

**Comment by umahatokula** [comment]
Just add this to User model

https://filamentphp.com/docs/3.x/panels/users#configuring-the-users-name-attribute
